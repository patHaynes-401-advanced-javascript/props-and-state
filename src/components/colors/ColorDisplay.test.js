import React from 'react';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

describe('ColorDisplay component', () => {
  it('renders ColorDisplay', () => {
    const wrapper = shallow(<ColorDisplay />);
    expect(wrapper).toMatchSnapshot();
  });
});
