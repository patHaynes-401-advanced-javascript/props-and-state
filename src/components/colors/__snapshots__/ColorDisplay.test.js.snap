// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorDisplay component renders ColorDisplay 1`] = `
<div
  style={
    Object {
      "backgroundColor": undefined,
      "height": "100px",
      "width": "100px",
    }
  }
/>
`;
